flickity/index.d.ts

```diff
@@ -161,84 +161,82 @@ declare class Flickity {
     * @param eventName name of event  (@see FlickityEvents class for filckity supported events)
     * @param callback callback funtion to execute when event fires
     */
-    on(eventname: string, callback: (eventt?: Event, cellElement?: Element) => any) : void;
+    on(eventname: string, callback: (event?: Event, pointer?: Element | Touch, cellElement?: Element, cellIndex?: number) => any): void;
```

Comment:
I've just changed the order of these, but these overloads don't really make sense -- they all take a string and a function, so how does the library know what kind of values to pass to the function? Can we use string literal types here? @clmcgrath